pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file:
+
+`composer init`
+
+- Create new project from an existing package:
+
+`composer create-project {{project_name}} {{target_path}} {{version}}`
```

Comment:
Also, I'm not entirely sure the distinction between a project, a package and the composer.json file is clear. This should be incorporated somewhere.